spec/features/eq_model_spec.rb

```diff
@@ -100,5 +100,14 @@
         it_behaves_like 'cannot see pending reservations'
       end
     end
+
+    context 'with requirement' do
+      it 'renders equipment with a requirement' do
+        @eq_model.requirements << FactoryGirl.create(:requirement)
+        sign_in_as_user @user
+        visit root_path
+        expect(page).to have_css(".equipment_title_link[title=\'#{@eq_model.name}\']")
```

Comment:
I didn't realize that you could use `title` in a CSS selector. I've normally seen this done this way:

```ruby
expect(page).to have_css('.equipment_title_link', text: @eq_model.name)
```

If that also works I think that would be my preference.

Reply:
FYI, the failed build was a style error in `spec/features/eq_model_spec.rb` - that should go away when you move the new test to its own file.

Reply:
The title selector is for the attribute the link has. The actual text that appears on the link can be truncated depending on how long the equipment model name is and I was worried about running up against that.